### PR TITLE
akka-cluster-tools compiler warnings as fatal errors 

### PR DESCRIPTION
**Pull request by johanandren**
Refs #26088

### CODE REVIEW

File: project/AkkaDisciplinePlugin.scala [hunk 1]
```diff
@@ -21,91 +21,71 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
...
   override lazy val projectSettings = disciplineSettings
 
-  val fatalWarningsFor = Set(
-    "akka-discovery",
-    "akka-distributed-data",
-    "akka-coordination",
-    "akka-protobuf"
-  )
+  val fatalWarningsFor =
+    Set("akka-discovery", "akka-distributed-data", "akka-coordination", "akka-protobuf", "akka-cluster-tools")
```
Comment (raboof) [inline project/AkkaDisciplinePlugin.scala]: I liked them on separate lines, but I guess this is the autoformatter kicking in... ok :(

**Review by raboof — APPROVED** [review]

**Comment by raboof** [comment]
Looks like some problems were still in `test:compile`?

**Comment by johanandren** [comment]
Yeah, forgot those!

File: project/AkkaDisciplinePlugin.scala (hunk 1 above)
Comment (johanandren) [reply project/AkkaDisciplinePlugin.scala]: The funky formatter is the culprit. (I agree on the LF:y one being nicer)

**Comment by johanandren** [comment]
Scala/javadoc validation failing

File: project/AkkaDisciplinePlugin.scala (hunk 1 above)
Comment (chbatey) [reply project/AkkaDisciplinePlugin.scala]: Resistance is futile

**Review by chbatey — APPROVED** [review]
LGTM

File: project/AkkaDisciplinePlugin.scala [hunk 2]
```diff
@@ -24,95 +24,78 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
     "akka-coordination",
     "akka-protobuf",
     "akka-stream-typed",
-    "akka-cluster-typed"
-  )
+    "akka-cluster-typed",
+    "akka - cluster - tools")
```
Comment (patriknw) [inline project/AkkaDisciplinePlugin.scala]: @johanandren isn't this wrong, with the spaces?

**Pull request merged**